tap_kustomer/sync.py

```diff
@@ -222,11 +211,19 @@ def sync_endpoint(client,  # pylint: disable=too-many-branches, too-many-stateme
         # set total_records and and last updated for pagination
         total_page = len(response.get('data'))
         if 'total' in response.get('meta'):
-            total_records = response.get('meta').get('total')
+            total_records = response['meta']['total']
+        if 'totalPages' in response.get('meta'):
+            total_page_count = response['meta']['totalPages']
         if len(response.get('data')) > 0:
             last_updated = response.get('data')[-1]['attributes']['updatedAt']
+
         # Get next link for end of pagination indication across all endpoints
+        # Note: sometimes the response does not return links even though
+        # there is supposedly more pages available. Manually requesting
+        # subsequent pages will return a 404 response code.
         next_page = response.get('links').get('next', None)
+        if next_page: 
+            next_url = '{}{}'.format(client.base_url, next_page)
```

Comment:
This is the fix. See on line 210 that we always fetch using `next_url`, but it was never being properly set.

Reply:
lgtm!